docs/develop/smart-contracts/guidelines/message-delivery-guarantees.mdx

```diff
@@ -0,0 +1,70 @@
+import ConceptImage from '@site/src/components/conceptImage'
+
+# Message delivery guarantees
+
+TON is an asynchronous blockchain with complex structure which is very different from other blockchains. In this regard, new developers have questions about low-level things in TON. In this article we will have a look at one of these related to message delivery.
```

Comment:
TON is an asynchronous blockchain with a complex structure which is very different from other blockchains. Because of this, new developers often have questions about low-level things in TON. In this article, we will have a look at one of these related to message delivery.